churchland_pipeline_python/__init__.py

```diff
@@ -1,14 +1,32 @@
 import datajoint as dj
-import os, glob
+import os, re, glob
 from os.path import dirname, basename, isfile, join
 from dotenv import load_dotenv, find_dotenv
 
 # load env file
-load_dotenv(join(dirname(__file__), '..', '.env')) 
+load_dotenv(join(dirname(__file__), '..', '.env'))
```

Comment:
this is not working if pip installed, so maybe add a try except here to make sure that when people pip install, it still works